portal/src/main/webapp/WEB-INF/jsp/global/frontend_config.jsp

```diff
@@ -13,6 +13,7 @@ window.legacySupportFrontendConfig = {
     genomeNexusApiUrl : '<%=GlobalProperties.getGenomeNexusApiUrl()%>',
     showCivic : <%=GlobalProperties.showCivic()%>,
     showHotspot : <%=GlobalProperties.showHotspot()%>,
+    showTwitter: '<%=GlobalProperties.showTwitterBookmark()%>',
```

Comment:
@Tamba32 @alisman remove quotes there to pass as boolean instead of string

Reply:
@inodb @alisman thanks!